recipes/protobuf/all/conanfile.py

```diff
@@ -48,20 +52,27 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
-            if self.settings.os == "Windows" and self.settings.compiler in ["Visual Studio", "clang"] and "MT" in self.settings.compiler.runtime:
+            if str(self.settings.compiler.get_safe("runtime")) in ["MT", "MTd", "static"]:
```

Comment:
why did it change?! https://github.com/conan-io/conan/blob/03b3d71890c537df1154b3d04cc35ed2654a5894/conans/client/conf/__init__.py#L91 

Nice catch you two!

Reply:
It is a new compiler setting, so strictly speaking it isn't changed :)